tracer/test/snapshots/AspNetMvc4Tests.Integrated.WithFF.__path=_Home_BadRequest_statusCode=500.verified.txt

```diff
@@ -14,11 +14,15 @@
 System.Exception: Oops, it broke.
 at Samples.AspNetMvc4.Controllers.HomeController.BadRequest(),
       error.type: System.Exception,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
+      http.route: /home/badrequest,
```

Comment:
I am so perplexed, this is an error that I was thinking only occurred in the `WithExpansion` scenario but I guess it also happens with the `WithFF` scenario 😵‍💫 There's a lot going on so I'm not sure if we have to solve this in this PR, but we should solve this before the next release